### DISCUSSION TITLE
Octoprint serial mismatch when preheating

### DISCUSSION THREAD

**Discussion by AxionHusky** [Jyers/Marlin#535] [Q&A] [answered]
![photo_2021-04-19_19-05-10](https://user-images.githubusercontent.com/82785211/115210738-3ee46100-a142-11eb-9948-52bd22196d7c.jpg)
I have discovered a rather interesting issue after installing the firmware. If I preheat the printer through it's own menu and then attempt to connect to it/send a file via octoprint, I get this from Octoprint: 

"Offline (Error: SerialException: 'device reports readiness to read but returned no data (device disconnected or multiple access on port?)' @ comm.py:_readline:3831)"

Occasionally the printer will also display a message on its screen saying it needs to be restarted when this occurs. I have determined that this issue only seems to arise when I preheat the printer.

Ender 3 V2, 4.2.2 board with no other additions.

**Comment by Tuncay-Ayhan** [comment]
Looks related to https://github.com/Jyers/Marlin/issues/510

Just wanted to point that out.

**Comment by Jyers** [comment, marked as answer]
Check this out to disable M112 from octoprint https://github.com/Jyers/Marlin/wiki/OctoPrint-Settings
